<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>689</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,2,3,6">
     <item row="8" column="0" rowspan="2">
      <widget class="QLabel" name="sex_lbl">
       <property name="text">
        <string>Пол:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="name_lbl">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QRadioButton" name="male_rb">
       <property name="text">
        <string>Мужской</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="firstname_le">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="secondname_le">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="middlename_lbl">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="surname_lbl">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="middlename_le">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QDateEdit" name="bitrh_date">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Образование:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="birth_date_lbl">
       <property name="text">
        <string>Дата рождения</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QRadioButton" name="female_rb">
       <property name="text">
        <string>Женский</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QComboBox" name="diploma_cmb">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>среднее профессиональное</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>неоконченное высшее</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>высшее магистр</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>высшее бакалавр</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>высшее специалист</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="0" colspan="3">
      <widget class="QLabel" name="exp_date_lbl">
       <property name="text">
        <string>Опыт работы:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="3">
      <widget class="QDateEdit" name="exp_date">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="3">
      <widget class="QLabel" name="cat_lbl">
       <property name="text">
        <string>Категория профессиональной деятельности:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="3">
      <widget class="QLineEdit" name="cat_le">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="3">
      <widget class="QLabel" name="prof_lbl">
       <property name="text">
        <string>Профессия:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="3">
      <widget class="QLineEdit" name="prof_le">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Номер и серия паспорта</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Номер контракта (создаётся автоматически)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QDateEdit" name="dateEdit_3">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Дата подписания контракта:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDateEdit" name="dateEdit_4">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Срок действия контракта:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Телефон сотрудника</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Адрес сотрудника</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="10">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="3" rowspan="7">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="18" column="0" colspan="3">
      <widget class="QLabel" name="position_lbl">
       <property name="text">
        <string>Должность</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="3">
      <widget class="QComboBox" name="position_cmb">
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
